.github/workflows/openscap.yml

```diff
@@ -16,16 +16,25 @@ jobs:
 
     steps:
       - name: Install dependencies
-        run: dnf install -y tar gcc git-core go openscap scap-security-guide
+        run: dnf install -y tar gcc git-core go openscap gh
 
       - name: Check out code into the Go module directory
         uses: actions/checkout@v4
 
+      - name: Download & extract latest ssg
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run:  |
+         gh release download --repo ComplianceAsCode/content --pattern "*.zip" --output /tmp/ssg.zip
```

Comment:
Waiting for feedback from the OpenSCAP team on this.  It's a simpler approach which allows us to use the upstream rather than rpms in upstream.